### PR TITLE
Mid-point tags/button

### PR DESCRIPTION
**Pull request by huwshimi**
Added tags and show more button to the mid-point.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- view `http://0.0.0.0:6543/:flags:/react/?midpoint=`
- check that there are tags and a "Show more" button

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3065/
Test PASSed.

File: jujugui/static/gui/src/app/components/mid-point/_mid-point.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .mid-point {
     color: $cool-grey;
-    padding: 30px 20px;
+    padding: 30px 20px 20px 20px;
```
Comment (anthonydillon) [inline jujugui/static/gui/src/app/components/mid-point/_mid-point.scss]: Trival but you dont need the 4th param here

**Comment by hatched** [comment]
:+1: QA :ok: !

**Comment by anthonydillon** [comment]
The tags should be orange on hover like on jujucharms. Also the commas should be `#333` ($cool-grey). :+1: with the changes mentioned.

**Comment by makyo** [comment]
Looks good! :+1: :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**